src/challenges/defi-challenges/simple-lending.mdx

```diff
@@ -0,0 +1,244 @@
+---
+name: Simple Lending
+index: 3
+level: 3
+website: "https://github.com/Turupawn/LendingChallengeLevelUp"
+summary: Understand the inner workings of Aave by completing the Lending challenge.
+heroImage: "/images/level-up-preview.png"
+labels: ["DeFi"]
+---
+
+import ContentCopy from "src/components/ContentCopy";
+
+# Simple Lending
+
+If you haven't completed the Token Swap challenge yet, check it out [here](https://www.levelup.xyz/challenges/simple-token-swap)!
+
+Let's take a step further and dive into another core DeFi primitive, lending. Aave is one of the top protocols, securing billions in assets. Let's learn how to earn yield by lending through smart contracts.
+
+In this challenge, you'll implement a contract that deposits into Aave pools on your behalf. You'll also be able to withdraw your supplied tokens along with the earned rewards.
+
+## Objective
+
+Your task is to:
+1. Develop your own contract with `deposit` and `withdraw` functionalities based on the provided base contract.
+2. Deploy it to Scroll Sepolia testnet.
+3. And finally verify it.
+ 
+If you need help with using a smart contract framework for completing this challenge, the [Level Up: Build with Foundry](https://www.levelup.xyz/content/level-up-foundry) guide might be a helpful start!
+
+If you get stuck, feel free to ask for help in [Level Up Telegram group](https://t.me/+PdNbk5milo1mMTAy).
+
+<ContentCopy code={`// SPDX-License-Identifier: MIT
```

Comment:
Remove ContentCopy component and can use the standard format writing code in markdown.